### PR TITLE
Allow setting workflow run timeout

### PR DESCRIPTION
**Pull request by feedmeapples**
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: cli/flags.go [hunk 1]
```diff
@@ -32,211 +32,212 @@ import (
...
+	FlagWorkflowIDReusePolicy             = "workflowidreusepolicy"
+	FlagWorkflowIDReusePolicyAlias        = FlagWorkflowIDReusePolicy + ", wrp"
+	FlagCronSchedule                      = "cron"
+	FlagWorkflowType                      = "workflow-type"
+	FlagWorkflowTypeWithAlias             = FlagWorkflowType + ", wt"
+	FlagWorkflowStatus                    = "status"
+	FlagWorkflowStatusWithAlias           = FlagWorkflowStatus + ", s"
+	FlagWorkflowExecutionTimeout          = "execution-timeout"
+	FlagWorkflowExecutionTimeoutWithAlias = FlagWorkflowExecutionTimeout + ", et"
+	FlagWorkflowRunTimeout                = "run-timeout"
```
Comment (feedmeapples) [inline cli/flags.go:74]: for reviewers, ^ that's the actual change

Comment (yiminc) [reply cli/flags.go:74]: why does it not have a shorter alias as others?

File: cli/flags.go [hunk 2]
```diff
@@ -310,7 +311,11 @@ var flagsForRunWorkflow = []cli.Flag{
 		Required: true,
 	},
 	&cli.IntFlag{
-		Name:  FlagExecutionTimeoutWithAlias,
+		Name:  FlagWorkflowRunTimeout,
+		Usage: "Single workflow run timeout in seconds",
+	},
+	&cli.IntFlag{
+		Name:  FlagWorkflowExecutionTimeoutWithAlias,
 		Usage: "Execution start to close timeout in seconds",
```
Comment (yiminc) [inline cli/flags.go:319]: may need a better doc to differentiate this with the single run timeout.

**Review by yiminc — APPROVED** [review]

File: cli/flags.go (hunk 2 above)
Comment (feedmeapples) [reply cli/flags.go]: updated the description. Also i want to an entire revamp of the help, including adding usage examples https://github.com/temporalio/proposals/pull/34

File: cli/flags.go (hunk 1 above)
Comment (feedmeapples) [reply cli/flags.go:74]: added temporarily for reference. These short versions won't work with urfave/cli v2 due to the api change. I have a task to adapt these to urfave/cli v2

**Pull request merged**